CMakeLists.txt

```diff
@@ -207,7 +228,7 @@ if(WIN32)
 else(WIN32)
     # build shared libraries
     set(NG_LIB_TYPE SHARED)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14")
```

Comment:
I think about raising the minimum CMake version to 3.1 and using
[CXX_STANDARD](https://cmake.org/cmake/help/v3.1/prop_tgt/CXX_STANDARD.html#prop_tgt:CXX_STANDARD) instead.

Reply:
is 14 actually really necessary? I think that was my problem.